webapp/components/Group/GroupForm.vue

```diff
@@ -169,18 +222,18 @@ export default {
         name: name || '',
         slug: slug || '',
         groupType: groupType || '',
-        about: about || '',
+        goal: goal || '',
```

Comment:
`goal` is called `about` in the database. So I would named it `about`…

Reply:
is changed